capstone/src/main/webapp/scripts/poll.js

```diff
@@ -5,8 +5,8 @@ let maxVotes;
 let topChallenge = null;
 const NO_CHALLENGES =
   "No current challenges. Submit a suggestion in the poll and mark checkbox for challenge to be posted. The challenge will be updated weekly based on top voted poll option.";
-let groupId;
 
+let groupId;
 function getGroupId() {
   groupId = window.location.search.substring(1).split("=")[1];
 }
```

Comment:
I think we can remove the group id related logic in this file. Since we are not namespacing the JS files, we're able to reference other files' functions without needing to import them. I see you added a similar getId() method in group.js, which we can reference instead of this one (I would also change the name to the same as this one so it makes more sense when you use it outside of group.js). Additionally, since group_id is a global variable, you'll similarly be able to use that across other files (though similarly, change the name to groupId. Since that's the name in poll.js you'll just need to change the references in group.js).